charts/victoria-metrics-k8s-stack/values.yaml

```diff
@@ -0,0 +1,449 @@
+    ##
+    jobLabel: node-exporter
+  extraArgs:
+    - --collector.filesystem.ignored-mount-points=^/(dev|proc|sys|var/lib/docker/.+|var/lib/kubelet/.+)($|/)
+    - --collector.filesystem.ignored-fs-types=^(autofs|binfmt_misc|bpf|cgroup2?|configfs|debugfs|devpts|devtmpfs|fusectl|hugetlbfs|iso9660|mqueue|nsfs|overlay|proc|procfs|pstore|rpc_pipefs|securityfs|selinuxfs|squashfs|sysfs|tracefs)$
+
+  vmServiceScrape:
+    # wheter we should create a service scrape resource for node-exporter
+    enabled: true
+
+    # spec for VMServiceScrape crd 
+    # https://github.com/VictoriaMetrics/operator/blob/master/docs/api.MD#vmservicescrapespec
+    spec:
+      jobLabel: jobLabel
+
+
+
+# kube-state-metrics dependency chart configuration. For possible values refer to https://github.com/kubernetes/kube-state-metrics/blob/master/charts/kube-state-metrics/values.yaml
+kube-state-metrics:
+  enabled: true
+  ## all values for kube-state-metrics helm chart can be specified here
+
+  # spec for VMServiceScrape crd 
+  # https://github.com/VictoriaMetrics/operator/blob/master/docs/api.MD#vmservicescrapespec
+  vmServiceScrape:
+    spec: {}
+
+  #TODO: selector override for kube-state-metrics deployed separatelly
+
+
+
+### Service Monitors
+## Component scraping the kubelets
+kubelet:
+  enabled: true
+
+  ## Enable scraping /metrics/cadvisor from kubelet's service
+  cadvisor: true
+  ## Enable scraping /metrics/probes from kubelet's service
+  probes: true
+  # spec for VMNodeScrape crd 
+  # https://github.com/VictoriaMetrics/operator/blob/master/docs/api.MD#vmnodescrapespec
+  spec:
+    scheme: "https"
+    honorLabels: true
+    interval: "30s"
+    scrapeTimeout: "5s"
+    tlsConfig:
+      insecureSkipVerify: true
+      caFile: "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+    bearerTokenFile: "/var/run/secrets/kubernetes.io/serviceaccount/token"
+    relabelConfigs:
+      - action: labelmap
+        regex: __meta_kubernetes_node_label_(.+)
+      - sourceLabels: [__metrics_path__]
+        targetLabel: metrics_path
+      - targetLabel: "job"
+        replacement: "kubelet"
+
+
+    
+## Component scraping the kube api server
+kubeApiServer:
+  enabled: true
+  # spec for VMServiceScrape crd 
+  # https://github.com/VictoriaMetrics/operator/blob/master/docs/api.MD#vmservicescrapespec
+  spec:
+    endpoints:
+    - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+      # bearerTokenSecret:
+      #   key: ""
+      port: https
+      scheme: https
+      tlsConfig:
+        caFile: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+        serverName: kubernetes
+    jobLabel: component
+    namespaceSelector:
+      matchNames:
+      - default
+    selector:
+      matchLabels:
+        component: apiserver
+        provider: kubernetes
+
+
+
+## Component scraping the kube controller manager
+kubeControllerManager:
+  enabled: true
```

Comment:
is it better to disable by default?

Reply:
i would go with default "true" and adding note to docs that for managed kubernetes like EKS,GKE you need to disable it